### PR TITLE
Add /tasks route to 2.0 API

### PR DESCRIPTION
**Pull request by geoff-reid**
Added Southbound /tasks rout to the 2.0 API.

Right now all swagger routes, including the new tasks route, appear on both the southbound and northbound interfaces.  I am working on changes that make the routes available only on the correct interfaces.  This will be a different pull request.

### CODE REVIEW

**Comment by jlongever** [comment]
test this please

**Comment by brianparry** [comment]
:+1:

File: static/monorail-2.0-sb.yaml [hunk 1]
```diff
@@ -0,0 +1,152 @@
+swagger: "2.0"
+info:
+  version: "0.0.1"
+  title: RackHD 2.0 Southbound
```
Comment (jlongever) [inline static/monorail-2.0-sb.yaml]: Is the plan to stick with the 1.1 paradigm of keeping these routes split over applying roles and access rules?

**Comment by abildw** [comment]
+1

**Comment by jlongever** [comment]
I suppose the answer is no then.

**Pull request merged**